src/YawPitchControl/YawPitchControl.js

```diff
@@ -228,6 +231,16 @@ const YawPitchControl = class YawPitchControl extends Component {
 			}
 		}
 
+		if (keys.some(key => key === "useGyro") && this.axesTiltMotionInput) {
+			const useGyro = this.options.useGyro;
+
+			if (useGyro === GYRO_MODE.YAWPITCH) {
+				this.axes.connect(["yaw", "pitch"], this.axesTiltMotionInput);
```

Comment:
@happyhj Is this not problem to connect again if already connected?

Reply:
If the useGyro value changes, the row will be executed, so there is no chance of duplicate execution.